modules/api-gateway/variables.tf

```diff
@@ -0,0 +1,127 @@
+variable "name" {
+  type    = string
+  default = "api-gw"
+}
+
+variable "endpoint_config_type" {
+  type    = string
+  default = "REGIONAL"
+}
+
+variable "stage_name" {
+  type    = string
+  default = "api-stage"
+}
+
+variable "rest_api_id" {
+  type    = string
+  default = ""
+}
+
+variable "open_api_path" {
+  type    = string
+  default = ""
+}
+
+variable "pgp_key" {
+  description = "Either a base-64 encoded PGP public key, or a keybase username in the form `keybase:username`. Used to encrypt password and access key. `pgp_key` is required when `create_iam_user_login_profile` is set to `true`"
+  type        = string
+  default     = ""
+}
+
+variable "iam_username" {
+  description = "username of newly created IAM user"
+  type        = string
+  default     = "api-gw-user"
+}
+
+variable "policy_name" {
+  description = "API Gateway policy name"
+  type        = string
+  default     = "api-gw-policy"
+}
+
```

Comment:
please cleanup for 
commented out variables,
blank new lines

also remove (or if it is important to have move into examples folder and have link to it) the file src/sample.json.tpl as it seems this is not in use anymore

in README.md replace module source from "../terraform-aws-modules/modules/api-gateway" to one from terraform repository